src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -294,7 +316,62 @@ void FullyConnected::prepareParams() {
     NodeDesc *selected_pd = getSelectedPrimitiveDescriptor();
     if (selected_pd == nullptr)
         IE_THROW() << "Preferable primitive descriptor is not set for node " << getName() << ".";
-
+#ifdef OV_CPU_WITH_MLAS
+    if (useMlas) {
+        auto prepareMLASWeight = [&](const int64_t M, const int64_t N, const int64_t K) {
+            if (!getParentEdgeAt(1)->getParent()->isConstant())
+                IE_THROW() << "Weight input is not const for node " << getName() << ".";
+            auto edgeMem = getParentEdgeAt(1)->getMemoryPtr();
+            std::string format = "gemm_mlas_" + std::to_string(N) + "_" + std::to_string(K);
+            if (!edgeMem)
+                IE_THROW() << "Cannot get const weights edgeMem for node " << getName() << ".";
+            auto packedBsize = ov_sgemm_pack_get_size(N, K);
+            MemoryPtr ptr;
+            auto create = [&] () {
+                float* weightPtr = reinterpret_cast<float*>(getParentEdgeAt(1)->getMemoryPtr()->GetPtr());
+                size_t ldb = K;
+                MemoryPtr _ptr = std::make_shared<Memory>(getEngine());
+                _ptr->Create(intel_cpu::DnnlBlockedMemoryDesc(Precision::FP32, intel_cpu::Shape{1, packedBsize / sizeof(float)}));
```

Comment:
Two questions here:
1. What is the unit of the packedBsize (bytes, elements ?). I assume that elements number is implied. Thus, suggest changing the `ov_sgemm_pack_get_size` call semantics so that it returns the size in bytes.
2. Why tensor of rank 2 with the leading dimension 1 is used?

Reply:
1. ov_sgemm_pack_get_size return the bytes of needed memory, let me comment it there
2. I think the memory shape [1, packedBsize ] could be reduced to [packedBsize] 
Thanks!